yb-voyager/cmd/exportData.go

```diff
@@ -362,14 +363,17 @@ func exportData() bool {
 
 		if changeStreamingIsEnabled(exportType) {
 			log.Infof("live migration complete, proceeding to cutover")
-			if isTargetDBExporter(exporterRole) {
+			msr, err := metaDB.GetMigrationStatusRecord()
+			if err != nil {
+				utils.ErrExit("get migration status record: %v", err)
+			}
+			if isTargetDBExporter(exporterRole) && !msr.UseLogicalReplicationYBConnector {
```

Comment:
shall we drop replication slot here as well? Do you see any concerns now that we sleep?